.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
Can we make this monthly or does one of you want to look at it every week eventually ? Asking because I already receive a lot of dependabot notifications.

Reply:
Sure :+1: I can't agree more about DB notifications :stuck_out_tongue: